src/lib/stores/wallet/wallet.store.ts

```diff
@@ -59,9 +59,14 @@ export default (() => {
     if (!browser) throw new Error('Can only connect client-side');
 
     const instance = await web3Modal.connect();
-
     const provider = new ethers.providers.Web3Provider(instance);
 
+    if (!_networkIsSupported(await provider.getNetwork())) {
```

Comment:
Keep in mind that if you try to do `AddressAppClient.create(unsupportedProvider)` it will throw an `unsupportedNetwork` `DripsError`. Just saying this for the "drips store".